Production/crab/config/2016/DY.txt

```diff
@@ -1,23 +1,26 @@
 EventAwareLumiBased
```

Comment:
in the updated txt format, an indication of job splitting algorithm is removed (i.e. remove line `EventAwareLumiBased`). See 2017 configs for the reference